bot/resources/evergreen/py_topics.yaml

```
@@ -23,6 +23,18 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Is python your first programming language? If not, what is it?
+    - What's your favourite aspect of python development? (Backend, web dev, game dev, machine learning, ai, etc.)
+    - In what ways has python discord helped you with python?
```

```suggestion
    - In what ways has Python Discord helped you with Python?
```
And other cases of `python` -> `Python`